ipyleaflet/leaflet.py

```diff
@@ -744,6 +744,21 @@ class ImageOverlay(RasterLayer):
     # Options
     attribution = Unicode().tag(sync=True, o=True)
 
+class SVGOverlay(ImageOverlay):
+    """SVGOverlay class.
+
+    SVG Image layer from a local or remote image file.
+
+    Attributes
+    ----------
+    el: string, default ""
+        svg element.
+    """
+
+    _view_name = Unicode('LeafletSVGOverlayView').tag(sync=True)
+    _model_name = Unicode('LeafletSVGOverlayModel').tag(sync=True)
+
+    el = Unicode().tag(sync=True)
```

Comment:
Suggestion: `value` instead of `el`? We use `value` in other places for ipywidgets and ipyleaflet.